source/sgp_mode/SGPWorld.h

```diff
@@ -45,6 +45,8 @@ class SGPWorld : public SymWorld {
   emp::Ptr<SyncDataMonitor<double>> data_node_sym_stolen;
   emp::Ptr<SyncDataMonitor<double>> data_node_sym_earned;
   emp::vector<emp::DataMonitor<size_t>> data_node_host_tasks;
+  emp::vector<emp::DataMonitor<size_t>> data_node_host_largest_square;
```

Comment:
Do these only track the largest square? Or all square counts? It should be named to accurately reflect what it does